gm4_vertical_rails/data/vertical_rails/tags/entity_types/minecart.json

```diff
@@ -0,0 +1,10 @@
+{
+    "values":[
+        "minecraft:minecart",
+        "minecraft:chest_minecart",
+        "minecraft:hopper_minecart",
+        "minecraft:furnace_minecart",
+        "minecraft:spawner_minecart",
+        "minecraft:tnt_minecart"
```

Comment:
This list should now include command_block_minecart since liquid minecarts are that type :)

Reply:
I though i added that one, i have done it now